server/service/src/invoice_line/inbound_shipment_line/insert/mod.rs

```diff
@@ -23,8 +27,8 @@ pub struct InsertInboundShipmentLine {
     pub sell_price_per_pack: f64,
     pub expiry_date: Option<NaiveDate>,
     pub number_of_packs: u32,
-    pub total_before_tax: f64,
-    pub tax: Option<f64>,
+    pub total_before_tax: Option<f64>,
+    pub tax: Option<ShipmentTaxUpdate>,
```

Comment:
I think it makes sense for insert to have just Option<f64> (there is no need to 'unset' it for inserts)